README.rst

```diff
@@ -51,6 +51,20 @@ the plugin to identify untested templates.
 If you get a :code:`django.core.exceptions.ImproperlyConfigured` error,
 you need to set the :code:`DJANGO_SETTINGS_MODULE` environment variable.
 
+If you get :code:`django_coverage_plugin.plugin.DjangoTemplatePluginException: Template debugging must be enabled in settings`, change your TEMPLATES setting:
+
+.. code-block:: python
+
+    DEBUG = True
```

Comment:
In my local testing this top-level Django `DEBUG` setting isn't required, only the template debug setting below is needed so `TEMPLATES ... 'OPTIONS': { 'debug': True }` *does* seem to work even with `DEBUG=False`.

Given it is preferable to run tests in an as-close-to-prod-as-possible configuration, perhaps remove this line & set `'debug': True,` below? (or `'debug': VAR_WHICH_IS_TRUE_IN_TESTING_ONLY` to avoid enabling debug templates in prod?)